src/AppBundle/Menu/MenuBuilder.php

```diff
@@ -83,27 +184,19 @@ public function createUserMenu(array $options)
     }
 
     /**
-     * @param $options
-     * @return \Knp\Menu\ItemInterface
-     * @throws \InvalidArgumentException
+     * @return bool
      */
-    public function createSideMenu($options)
+    private function isAdmin()
     {
-        $menu = $this->factory->createItem('root');
-        $menu->setChildrenAttribute('class', 'side-nav');
-        $menu->setChildrenAttribute('id', 'mobile-nav');
-
-        $user = $this->createUserMenu($options);
-        $nav = $this->createNavMenu();
-
-        foreach ($nav->getChildren() as $child) {
-            $menu->addChild($child->copy());
-        }
+        return $this->checker->isGranted('ROLE_ADMIN');
+    }
 
-        foreach ($user->getChildren() as $child) {
-            $menu->addChild($child->copy());
-        }
+    /**
+     * @return bool
+     */
+    private function isLoggedIn()
+    {
+        return $this->checker->isGranted('ROLE_ADMIN');
```

Comment:
Accidentally true, but if we had more roles isLoggedIn would check against different roles.
So while the methods have the same body they are not semantically equal.